app/Config/core.php

```diff
@@ -215,19 +215,19 @@
  * the cake shell command: cake schema create Sessions
  *
  */
-	Configure::write('Session', array(
-		'defaults' => 'php'
-	));
+    Configure::write('Session', array(
+        'defaults' => 'php'
+    ));
```

Comment:
__[staging][code_sniffer]__
* Closing parenthesis not aligned correctly; expected 32 space(s) but found 4
* Closing parenthesis of a multi-line function call must be on a line by itself